api/v1alpha1/groupversion_info.go

```diff
@@ -0,0 +1,35 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1alpha1 contains API Schema definitions for the obsapi v1alpha1 API group
+// +kubebuilder:object:generate=true
+// +groupName=obs-api.observatorium.io
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/scheme"
+)
+
+var (
+	// GroupVersion is group version used to register these objects
+	GroupVersion = schema.GroupVersion{Group: "obs-api.observatorium.io", Version: "v1alpha1"}
```

Comment:
I don't think `obs-api` is a very useful group name. All `observatorium.io` API groups will probably be `obs ` and they are all already `API` by definition. `obs-api` doesn't actually add any information IMO. Also, `obs` is not really a meaningful/legible string for most people, me included :)

maybe simply "core" would be better

Reply:
@brancz Any thoughts on what should be the group name?
Isn't "core" too generic?

Reply:
IMO it is more meaningful than `obs-api`. An API group named `api` is the most generic API group i can think of. I'd prefer to put some thought into this now rather than have to change API groups later down the road

Reply:
Done